src/Webapi/Canvas/Webapi__Canvas__Path_Common.res

```diff
@@ -0,0 +1,81 @@
+module type ModuleWithT = {
```

Comment:
The style used in this project is `Impl` and `T.t`, with the module type and module in one declaration

https://github.com/TheSpyder/rescript-webapi/blob/87270bd837a826c4ba29c7a8fe91ce7ebf04813a/src/Webapi/Dom/Webapi__Dom__Event.res#L1-L5

https://github.com/TheSpyder/rescript-webapi/blob/18d9d553201d575fa8e194d3370298b556b28747/src/Webapi/Dom/Webapi__Dom__PointerEvent.res#L4-L6